### PR TITLE
exclude experimental from ci

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

## Checklist

- [ ] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by ktangsali — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -85,8 +85,8 @@ fixable = ["I"]
 # and `S311` (random number generators)
 ignore = ["E501", "S311"]
 
-# Exclude the examples folder 
-exclude = ["examples"]
+# Exclude the examples and experimental folders 
+exclude = ["examples", "experimental"]
```
Comment (NickGeneva) [inline pyproject.toml:89]: shouldn't this be modulus/experimental ?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -17,7 +17,9 @@ repos:
     rev: 22.10.0
     hooks:
     - id: black
-      exclude: "docs"
+      exclude:
+      - "docs"
+      - "experimental"
```
Comment (NickGeneva) [inline .pre-commit-config.yaml:22]: `experimental` or `modulus/experimental`

**Comment by NickGeneva** [comment]
Hmmm precommit config not really working?

**Comment by mnabian** [comment]
> Hmmm precommit config not really working?

I think i know what's going on. let me fix it really quick

**Review by NickGeneva — APPROVED** [review]

**Comment by NickGeneva** [comment]
Approved, thank you for edits!

**Pull request merged**